02week/ticTacToe.js

```diff
@@ -24,23 +24,56 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  // Possible horizontal wins
+  let winArrH = board[0][0];
+  if ((board[0][0] == playerTurn && board[0][1] == playerTurn && board[0][2] == playerTurn) || (board[1][0] == playerTurn && board[1][1] == playerTurn && board[1][2] == playerTurn) || (board[0][0] == playerTurn && board[0][1] == playerTurn && board[0][2] == playerTurn)) {
```

Comment:
@rmcbride2121 
When lines get this long, you should break them into multiple lines for readability